sdk/storage/azure-storage-blob/README.md

```diff
@@ -9,159 +9,175 @@ Blob storage is ideal for:
 
-If you need to create a new resource group:
+### Create a storage account
+If you wish to create a new storage account, you can use the
+[Azure Portal](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal),
+[Azure PowerShell](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell),
+or [Azure CLI](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli):
 
 ```bash
+# Create a new resource group to hold the storage account -
+# if using an existing resource group, skip this step
 az group create --name my-resource-group --location westus2
-```
-
-Create your storage account:
 
-```bash
-az storage account create -n mystorageaccountname -g my-resource-group
+# Create the storage account
+az storage account create -n my-storage-account-name -g my-resource-group
 ```
 
+### Create the client
+The Azure Storage Blobs client library for Python allows you to interact with three types of resources: the storage
+account itself, blob storage containers, and blobs. Interaction with these resources starts with an instance of a
+[client](#clients). To create a client object, you will need the storage account's blob service endpoint URL and a
+credential that allows you to access the storage account:
 
-### Authenticate the client
-
-Interaction with Storage Blobs starts with an instance of the BlobServiceClient class. You need an existing storage account, its URL, and a credential to instantiate the client object.
-
-#### Get credentials
-
-To authenticate the client you have a few options:
-1. Use a SAS token string 
-2. Use an account shared access key
-3. Use a token credential from [azure.identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity)
+```python
+from azure.storage.blob import BlobServiceClient
 
-Alternatively, you can authenticate with a storage connection string using the `from_connection_string` method. See example: [Client creation with a connection string](#client-creation-with-a-connection-string).
+service = BlobServiceClient(account_url="https://<my-storage-account-name>.blob.core.windows.net/", credential=credential)
+```
 
-You can omit the credential if your account URL already has a SAS token.
+#### Looking up the endpoint URL
+You can find the storage account's blob service endpoint URL using the 
+[Azure Portal](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal),
+[Azure PowerShell](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell),
+or [Azure CLI](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli):
```

Comment:
Are these the right links? these go to the "Create a storage account" page and I don't see anything there about service endpoint URLs

Reply:
Good call. I will look for better links - these were in the original version, I just reformatted this section to add line breaks.